apps/api/src/authorization/mock-jwt-auth-guard.ts

```diff
@@ -0,0 +1,30 @@
+import { ExecutionContext, UnauthorizedException } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { CoscradUserWithGroups } from '../domain/models/user-management/user/entities/user/coscrad-user-with-groups';
+
+export class MockJwtAuthGuard extends AuthGuard('jwt') {
+    constructor(
+        private readonly testUserWithGroups: CoscradUserWithGroups | undefined,
+        private readonly isAuthOptional: boolean
+    ) {
+        super();
+    }
+
+    override handleRequest<TUser = any>(_: any, __: any, ___: any, ____: any, _____?: any) {
```

Comment:
Whoa!!

Reply:
I'll check if it's actually necessary to put the unused args here. It's really ugly.

Reply:
How about this
```ts
    override handleRequest<TUser = unknown>(
        _: unknown,
        __: unknown,
        ___: unknown,
        ____: unknown,
        _____?: unknown
    )
```

Reply:
But it was unnecessary- fixed it!